ember-core/shared/src/main/scala/org/http4s/ember/core/h2/H2Frame.scala

```diff
@@ -19,7 +19,11 @@ package org.http4s.ember.core.h2
 import cats.syntax.all._
 import scodec.bits._
 
+import scala.annotation.nowarn
+
 private[ember] sealed trait H2Frame
+
+@nowarn("msg=implicit numeric widening")
```

Comment:
I usually prefer `cat`, but I haven't had a ton of luck with that and Scala 3, and this seemed to work.